projects/core/build/tokens.ts

```diff
@@ -536,7 +561,7 @@ const aliases = {
       },
     },
     app: {
-      background: token(color.gray[50]),
+      background: token(color.construction[25]),
```

Comment:
Just checking: this effectively changes the background of our applications from #fafafa to #fcfdfd, is that intentional?

Reply:
Yes